scalafix/build.sbt

```diff
@@ -4,6 +4,10 @@ inThisBuild(
   List(
     organization := "org.http4s",
     version := outputVersion,
+    isSnapshot := {
+      if (outputVersion == "0.20.0") false
+      else isSnapshot.value
+    },
```

Comment:
This can go away after we close the Sonatype repo.  I was fighting the dynver plugin that gives us `isSnapshot`.

Reply:
I guess it may be because dynver expects to see the tag `v0.20.0` at zero distance from HEAD...

Reply:
I was also working on this before I committed.  I could just as well have hardcoded the publishTo and skipped the `isSnapshot` until we integrate this with the other plugins. :shrug: